lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
@jdolan `rest_base` is not about `post_tag` we have to support it for custom taxonomies too. If anything, this commit supports you argument, it is _getting rid_ of special casing the case.